openstf/pipeline/train_model.py

```diff
@@ -247,3 +247,17 @@ def train_pipeline_common(
         pj, validation_data
     ).generate_standard_deviation_data(model)
     return model, train_data, validation_data, test_data
+
+
+def get_model_age(trained_models_folder: str, pid: int) -> float:
```

Comment:
Do we still need this function? Can we not directly call the PersistentStorageSerializer(trained_models_folder).determine_model_age_from_pid(pid) instead?

Reply:
You're right, I'll remove this superfluous function